### DISCUSSION TITLE
whether your Docs pages are developed by Lowdefy?

### DISCUSSION THREAD

**Discussion by wantingjun** [lowdefy/lowdefy#651] [Questions and Help] [answered]
Hi,

Your project is interesting, it’s an useful tool for someone who is not familiar with React.js to build front-end pages. Recently, I try to use Lowdefy to build several apps, one of them is a little bit similar to your [tutorial Docs](https://docs.lowdefy.com/ ), so I am wondering whether your Docs are developed by Lowdefy (through writing YAML or JSON) rather than React? 

Thanks in advance for your help.

**Comment by SamTolmay** [comment, marked as answer]
Hi @wantingjun 

Thanks, I am glad you find it useful.

Yes, the docs are developed using Lowdefy. You can see our implementation here: https://github.com/lowdefy/lowdefy/tree/main/packages/docs

We have found it very useful, especially when documenting blocks since we can just use the blocks already in Lowdefy as examples.

We use references quite extensively to create templates. This ensures all the pages are the same, so, for example, when changing the footer, it is updated on all pages. You can read more about references here:  https://docs.lowdefy.com/lowdefy-schema (towards the bottom of the page).

The general page template can be found here: https://github.com/lowdefy/lowdefy/blob/main/packages/docs/templates/general.yaml.njk

Please feel free to get in contact with us if you have any other questions, either here, or on our [discord](https://discord.gg/WmcJgXt)
